README.md

```diff
@@ -1,291 +1,124 @@
 <a name="readme-top"></a>
```

Comment:
- [ ] Kindly, add a 'tests' section on your readme with specific instructions about how to correctly run your tests.  Kindly, make sure all of them pass. 👍